CIP-0054/README.md

```diff
@@ -243,9 +256,10 @@ In tandem with the additional metadata, we also define a standard for the Javasc
 
 ### Implementation Plan
 
-- [ ] Provide a reference implementation of this scheme, which illustrates both:
-  - [ ] a means of creating a "Smart NFT"
-  - [ ] a means of rendering it
+- [X] Provide a [reference](https://github.com/kieransimkin/libcip54) [implementation](https://github.com/kieransimkin/smartnftportal) of this scheme, which illustrates both:
+  - [X] [a means of creating a "Smart NFT"](https://nft-playground.dev/)
+  - [X] [a means of rendering it](https://clg.wtf/)
+  - [ ] Update this specification to match the new features added in the reference implementation.
```

Comment:
Does this item (box currently unticked) match what you've added in `## The Smart NFT toolchain`?  If so, the box should be ticked... if not, will you be adding the specification update in _this_ PR?

Reply:
The specification update is still pending - I'm really waiting until the API has settled down a bit more before I update the CIP, as it will be quite a major update and I only really want to have to do it once - currently there's still quite a lot of additions and tweaks happening to the API so it doesn't make sense to be constantly updating the CIP - the CIP does still give the broad essence of what's available - it's not incorrect, it's just missing a bunch of stuff that has been added since. For now you could just call these "undocumented features" ;) although many of them are actually documented in the examples section of the NFT playground.